reference/glossary.md

```diff
@@ -0,0 +1,445 @@
+### Monitoring stack
+
+The part of the Anbox Cloud stack that provides monitoring functionality.
+
+See https://discourse.ubuntu.com/t/install-the-monitoring-stack/17786.
+
+<a name="nagios"></a>
+### Nagios
+
+A tool for monitoring the status of critical infrastructure like networks, servers and applications.
+
+See [the Nagios website](https://www.nagios.org/).
+
+<a name="nats"></a>
+### Neural Autonomic Transport System (NATS)
+
+An open-source messaging system that the components of the streaming stack use to communicate.
+
+See [the NATS website](https://nats.io/).
+
+<a name="platform"></a>
+### Platform
+
+An abstraction layer that is provided by Anbox to access the hardware resources of the host system from the Android system. Anbox Cloud supports three platforms: null (without rendering), webrtc (WebRTC) and swrast (software rendering).
+
+See https://discourse.ubuntu.com/t/anbox-platforms/18733.
+
+<a name="prometheus"></a>
+### Prometheus
+
+An open-source application used for event monitoring and alerting, which records real-time metrics about system events.
+
+See [the Prometheus website](https://prometheus.io/).
+
+<a name="raw-container"></a>
+### Raw container
+
+A container that is created when launching an image. It runs the full Android system, without any additional apps installed.
+
+See [Application containers vs. raw containers](https://discourse.ubuntu.com/t/about-containers/17763#application-vs-raw).
+
+<a name="regular-container"></a>
+### Regular container
+
+A container that is launched from either an application or an image. It exists until it is deleted.
+
+See https://discourse.ubuntu.com/t/about-containers/17763.
+
+<a name="scrcpy"></a>
+### Scrcpy
+
+An open-source screen mirroring application that allows displaying and controlling Android devices from a desktop computer.
+
+See [the scrcpy project on GitHub](https://github.com/Genymobile/scrcpy).
+
+<a name="session"></a>
+### Session
+
+The interaction between a streaming client and the application container during streaming. A session contains, among other information, user data and application information and provides an entry point for both the client and the container to start the signalling process.
+
+See https://discourse.ubuntu.com/t/about-application-streaming/17769.
+
+<a name="snap"></a>
+### Snap
+
+A software package for a desktop, cloud or IoT application that is easy to install, secure, cross‐platform and dependency‐free.
+
+See [the Snapcraft website](https://snapcraft.io/).
+
+<a name="swrast"></a>
+### Software Rasterization (swrast)
+
+An LLVMpipe-based software rendering platform that is useful for visual tests but does not provide audio input/output.
+
+See https://discourse.ubuntu.com/t/anbox-platforms/18733.
+
+<a name="stream-agent"></a>
+### Stream agent
+
+The software running on a server equipped with a GPU connected to Anbox Cloud, which serves as an entry point that the stream gateway can connect to.
```

Comment:
It doesn't necessarily need to be a server with a GPU. The stream agent connects AMS to the stream gateway and allows distribution from the gateway to multiple independent AMS installations.